api/src/pcapi/core/offerers/repository.py

```diff
@@ -473,3 +474,41 @@ def find_venues_of_offerer_from_siret(siret: str) -> tuple[models.Offerer | None
         .all()
     )
     return offerer, venues
+
+
+def get_offerer_bank_accounts(offerer_id: int) -> models.Offerer:
+    """
+    Return an Offerer with its accounting data and related venues:
+
+    - Existing bank accounts (possibly none and only active & not (refused or without confirmation) ones)
+    - Linked venues to his bank accounts (possibly none and only current ones)
+    - Managed venues by the offerer (possibly none)
+    """
+
+    return (
+        models.Offerer.query.filter_by(id=offerer_id)
+        .outerjoin(finance_models.BankAccount)
+        .outerjoin(models.Venue)
+        .outerjoin(
+            models.VenueBankAccountLink,
+            sqla.and_(
+                finance_models.BankAccount.id == models.VenueBankAccountLink.bankAccountId,
+                sqla.func.upper(models.VenueBankAccountLink.timespan).is_(None),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏pourquoi tu as besoin du `upper` si tu fait `.is_(None)` après ?

Reply:
timespan possède une borne inférieur et une borne supérieur, je souhaite seulement effectuer ma jointure sur les liens avec une borné supérieur nulle (lien actuel) @lixxday